backend/app/routers/order.py

```
@@ -277,3 +237,63 @@ async def delete_order(
     db.delete(order)
     db.commit()
     return {"detail": ORDER_DELETED}
+
+
+@router.get("/orders/artisan")
+async def get_orders_for_artisan(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_orders_for_artisan`

(ANN201)